### ISSUE TITLE
Controller crashes if AS3 schema cannot be fetched

### ISSUE THREAD

**Issue by drennalls** [F5Networks/k8s-bigip-ctlr#1040]
#### Description
We were using `f5networks/k8s-bigip-ctlr:1.10` without issues, then upgraded to `1.11` and the controller is now crashing on startup because it cannot fetch the AS3 schema from github. The controller should not assume it has external network access.

#### Kubernetes Version
v1.14.6

#### Controller Version
1.11

#### BIG-IP Version
12.1.4

#### Diagnostic Information
debug output from controller pod when it dies..
```
15:17 $ ks logs -l app=f5-bigip-ctlr -f
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,646 urllib3.connectionpool DEBUG] Starting new HTTPS connection (1): 10.28.108.3:443
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,682 urllib3.connectionpool DEBUG] https://10.28.108.3:443 "GET /mgmt/tm/sys/ HTTP/1.1" 200 3409
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,687 f5_cccl DEBUG] F5CloudServiceManager initialize
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,687 f5_cccl.bigip DEBUG] BigIPProxy.__init__()
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,851 f5_cccl DEBUG] F5CloudServiceManager initialize
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,851 f5_cccl.bigip DEBUG] BigIPProxy.__init__()
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,863 __main__ DEBUG] config handler thread start
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,864 __main__ DEBUG] config handler woken for reset
2019/10/10 19:17:13 [DEBUG] [2019-10-10 19:17:13,870 __main__ DEBUG] loaded configuration file successfully
2019/10/10 19:17:13 [INFO] [2019-10-10 19:17:13,872 __main__ INFO] entering inotify loop to watch /tmp/k8s-bigip-ctlr.config391956280/config.json
2019/10/10 19:17:43 [DEBUG] error while fetching latest as3 schema : Get https://raw.githubusercontent.com/F5Networks/f5-appsvcs-extension/master/schema/latest/as3-schema.json: dial tcp 151.101.192.133:443: i/o timeout
2019/10/10 19:17:43 [DEBUG] ConfigWriter (0xc000276de0) received stop signal
2019/10/10 19:17:43 [INFO] ConfigWriter stopped: 0xc000276de0
2019/10/10 19:17:43 [DEBUG] [2019-10-10 19:17:43,009 __main__ DEBUG] config file /tmp/k8s-bigip-ctlr.config391956280/config.json changed - signalling bigip
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x11b8f5b]

goroutine 1 [running]:
main.fetchAS3Schema(0xc00036e000)
        /build/src/github.com/F5Networks/k8s-bigip-ctlr/cmd/k8s-bigip-ctlr/main.go:788 +0x10b
main.main()
        /build/src/github.com/F5Networks/k8s-bigip-ctlr/cmd/k8s-bigip-ctlr/main.go:743 +0xcde
```

**Comment by mdditt2000** [comment]
Created CONTCNTR-1154 for PM tracking. @drennalls we will be creating a patch release CIS 1.11.1 that has a flag to disable this.

**Comment by mdditt2000** [comment]
@trinaths can we go ahead and close this

**Comment by trinaths** [comment]
This issue is resolved in 1.12. Closing this issue.
